sklearn/manifold/spectral_embedding_.py

```diff
@@ -116,6 +116,13 @@ def _set_diag(laplacian, value):
     return laplacian
 
 
+def eigen_sign_flip(u):
```

Comment:
Please mark this as a private helper function by renaming it to `_eigen_sign_flip`. Also please add a small docstring to explain what is does and what is the motivation.